repos/system_upgrade/el7toel8/actors/repositoriesmapping/actor.py

```diff
@@ -12,26 +16,41 @@ class RepositoriesMapping(Actor):
 
     name = 'repository_mapping'
     consumes = ()
-    produces = (RepositoriesMap,)
+    produces = (RepositoriesMap, Report,)
     tags = (IPUWorkflowTag, ChecksPhaseTag)
+    file_path = '/etc/leapp/files/repomap.csv'
```

Comment:
I think this can be a variable inside "process", maybe being exposed for all actor is unnecessary.